integ-test/src/test/java/org/opensearch/sql/sql/MatchIT.java

```diff
@@ -36,6 +37,16 @@ public void match_in_having() throws IOException {
     verifyDataRows(result, rows("Bates"));
   }
 
+  @Test
+  public void missing_field_test() {
+    String query = StringUtils.format("SELECT * FROM %s WHERE match(invalid, 'Bates')", TEST_INDEX_ACCOUNT);
+    final RuntimeException exception =
+        expectThrows(RuntimeException.class, () -> executeJdbcRequest(query));
+    assertTrue(exception.getMessage()
+        .contains("can't resolve Symbol(namespace=FIELD_NAME, name=invalid) in type env") &&
+        exception.getMessage().contains("SemanticCheckException"));
```

Comment:
It'd be neater, if this was split into separate assertions.

As written, when `assertTrue` fails it can be for one of two reasons.

Reply:
Or use `assertAll`